src/components/DownloadButtonAndMenu.tsx

```diff
@@ -7,8 +6,18 @@ import {
 } from "@/components/ui/dropdown-menu"
 import { Button } from "@/components/ui/button"
 import { Download, ChevronDown } from "lucide-react"
+import { useDownloadJSON } from "@/hooks/use-download-json"
 
 export function DownloadButtonAndMenu({ className }: { className?: string }) {
+  const createBlobURL = (content: string) => {
```

Comment:
Generic methods like this shouldnt be put in the inner scope of a function, move to utils directory